src/registry/types.ts

```diff
@@ -4,6 +4,7 @@ export type ChainInfo = {
 	[x: string]: {
 		specName: string;
 		tokens: string[];
+		assetIds?: number[];
```

Comment:
I think this can just be `assetIds: number[]` since if its not a common good parachain it will just be empty